src/devices/LiquidLevelSwitch/PinMode.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.LiquidLevelSwitch
+{
+    /// <summary>
+    /// Liquid level liquid detection pin mode
+    /// </summary>
+    public enum PinMode
```

Comment:
I thought you were using System.Device.Gpio.PinMode... I'd remove this enum and just replace it with `[System.Device.Gpio.]PinMode liquidPresentPinState` or similar name